src/pint/utils.py

```diff
@@ -1213,27 +1329,29 @@ def mass_funct2(mp, mc, i):
 
     Parameters
     ----------
-    mp : Quantity
-        Pulsar mass, typically in `u.solMass`
-    mc : Quantity
-        Companion mass, typically in `u.solMass`
-    i : Angle
-        Inclination angle, in `u.deg` or `u.rad`
+    mp : :class:`astropy.units.Quantity`
+        Pulsar mass, typically in ``u.solMass``
+    mc : :class:`astropy.units.Quantity`
+        Companion mass, typically in ``u.solMass``
+    i : :class:`astropy.coordinates.Angle`
+        Inclination angle, in ``u.deg`` or ``u.rad``
 
     Returns
     -------
-    f_m : Quantity
-        Mass function in `u.solMass`
+    f_m : :class:`astropy.units.Quantity`
```

Comment:
In Parameters and Returns sections, for types of parameters/return values, you shouldn't need `:class:` and backticks, it should automatically get parsed as a type.